### PR TITLE
Added changelog and upgraded the versions for household, individual a…

### PR DESCRIPTION
**Pull request by kanishq-egov**
…nd service request

## Summary by CodeRabbit

- **New Features**
	- Introduced a `BOOLEAN` DataType for enhanced attribute definition capabilities in the service request framework.
	- Implemented fixes for `ExistentEntityValidator` across multiple health services modules.

- **Version Updates**
	- Updated service request module to version 1.0.1.
	- Updated household module to version 1.1.4.
	- Updated individual module to version 1.1.6.
	- Updated project module with validation for project start and end dates.
	- Updated stock module with enhancements to inventory flow.

- **Documentation**
	- Updated changelogs for service request, household, individual, project, referral management, and stock modules to reflect recent changes and version updates.

### CODE REVIEW

File: health-services/household/CHANGELOG.md [hunk 1]
```diff
@@ -1,7 +1,10 @@
 All notable changes to this module will be documented in this file.
 
+## 1.1.4 - 2024-08-29
+- Added `ExistentEntityValidator` fixes
+
 ## 1.1.3 - 2024-05-29
-- Integrated Core 2.9LTS
+- Integrated Core 2.9 LTS
```
[the remark this answers is not in the document]
Comment (kanishq-egov) [reply health-services/household/CHANGELOG.md:7]: irrelevant, closing this comment, 1.1.3 header is not a list.

File: health-services/individual/CHANGELOG.md [hunk 2]
```diff
@@ -1,5 +1,7 @@
 All notable changes to this module will be documented in this file.
 
+## 1.1.6 - 2024-08-29
```
[the remark this answers is not in the document]
Comment (kanishq-egov) [reply health-services/individual/CHANGELOG.md:3]: same as before

File: health-services/household/CHANGELOG.md [hunk 3]
```diff
@@ -1,7 +1,10 @@
 All notable changes to this module will be documented in this file.
 
+## 1.1.4 - 2024-08-29
+- Added `ExistentEntityValidator` fixes
```
[the remark this answers is not in the document]
Comment (kanishq-egov) [reply health-services/household/CHANGELOG.md:5]: suggestion have been, resolving this

File: core-services/service-request/CHANGELOG.md [hunk 4]
```diff
@@ -1,5 +1,8 @@
 All notable changes to this module will be documented in this file.
 
+## 1.0.1 - 2024-08-29
+- Added `BOOLEAN` DataType in `AttributeDefinition`
```
[the remark this answers is not in the document]
Comment (kanishq-egov) [reply core-services/service-request/CHANGELOG.md:5]: irrelevant, closing this

File: health-services/individual/CHANGELOG.md [hunk 5]
```diff
@@ -1,5 +1,8 @@
...
+
+ - Added `ExistentEntityValidator` fixes
```
[the remark this answers is not in the document]
Comment (kanishq-egov) [reply health-services/individual/CHANGELOG.md:5]: irrelevant, closing this, there is basically no changes here

File: health-services/individual/CHANGELOG.md [hunk 6]
```diff
@@ -1,5 +1,11 @@
 All notable changes to this module will be documented in this file.
 
+## 1.1.6 - 2024-08-29
+ ## 1.1.6 - 2024-08-29
```
[the remark this answers is not in the document]
Comment (kanishq-egov) [reply health-services/individual/CHANGELOG.md]: outdated, closing this

File: health-services/household/CHANGELOG.md [hunk 7]
```diff
@@ -1,7 +1,10 @@
 All notable changes to this module will be documented in this file.
 
+## 1.1.4 - 2024-08-29
```
[the remark this answers is not in the document]
Comment (kanishq-egov) [reply health-services/household/CHANGELOG.md]: outdated.

File: core-services/service-request/CHANGELOG.md [hunk 8]
```diff
@@ -1,5 +1,9 @@
 All notable changes to this module will be documented in this file.
 
+## 1.0.1 - 2024-08-29 
+
+- Added `BOOLEAN` DataType in `AttributeDefinition`
+
```
[the remark this answers is not in the document]
Comment (kanishq-egov) [reply core-services/service-request/CHANGELOG.md:6]: irrelevant, closing this

File: health-services/individual/CHANGELOG.md [hunk 9]
```diff
@@ -1,7 +1,11 @@
 All notable changes to this module will be documented in this file.
 
+## 1.1.6 - 2024-08-29 
+
+ - Added `ExistentEntityValidator` fixes
+
```
[the remark this answers is not in the document]
Comment (kanishq-egov) [reply health-services/individual/CHANGELOG.md:6]: irrelevant, closing this

File: health-services/household/CHANGELOG.md [hunk 10]
```diff
@@ -1,7 +1,12 @@
...
 
-## 1.1.3 - 2024-05-29
-- Integrated Core 2.9LTS
+## 1.1.4 - 2024-08-29 
+
+- Added `ExistentEntityValidator` fixes
+
+## 1.1.3 - 2024-05-29 
+
+- Integrated Core 2.9 LTS
```
[the remark this answers is not in the document]
Comment (kanishq-egov) [reply health-services/household/CHANGELOG.md:9]: irrelevant, closing this

**Comment by kanishq-egov** [comment]
@coderabbitai review

**Review by kavi-egov — APPROVED** [review]

**Review by kavi-egov — APPROVED** [review]

File: health-services/project/CHANGELOG.md [hunk 11]
```diff
@@ -13,6 +13,7 @@ All notable changes to this module will be documented in this file.
 - Upgraded PostgresSQL Driver version to 42.7.1
 - Upgraded Flyway base image version to 10.7.1 for DB Migration
 - Upgraded Flyway-Core to 9.22.3
+- Added `ExistentEntityValidator` fixes
```
[the remark this answers is not in the document]
Comment (kanishq-egov) [reply health-services/project/CHANGELOG.md]: irrelevant, closing this

**Pull request closed without merging**